onnx/backend/test/case/node/sequenceinsert.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def sequence_insert_reference_implementation(sequence, tensor, position=None):  # type: (np.ndarray, np.ndarray, np.ndarray) -> np.ndarray
```

Comment:
need to be careful with python objects: the list is passed by reference, so `sequence` gets updated as well. Can use `copy` to create a new sequence to be udpated.

Reply:
- since list doesn't have a .copy() function and I didn't want to introduce the copy library, I'm using the list() casting functionality to create a new copy.